pkg/reporting/cookbooks.go

```diff
@@ -258,23 +264,23 @@ func (cbs *CookbooksStatus) downloadCookbook(cookbookName, version string, analy
 
 	// by default we report only cookbooks that are being used by one or more nodes,
 	// but we also provide a way to report the opposite, that is, only unused cookbooks
-	if cbs.OnlyUnused {
+	if cbr.onlyUnused {
 		// report only unused cookbooks
 		if len(nodes) > 0 {
-			cbs.progress.Increment()
+			cbr.progress.Increment()
 			return
 		}
 	} else {
 		// report only cookbooks being used
 		if len(nodes) == 0 {
-			cbs.progress.Increment()
+			cbr.progress.Increment()
 			return
 		}
 	}
 
 	// do we need to analyze the cookbooks
-	if cbs.RunCookstyle {
-		err = cbs.Cookbooks.DownloadTo(cookbookName, version, cbs.CookbooksDir)
+	if cbr.RunCookstyle {
+		err = cbr.cookbooks.DownloadTo(cookbookName, version, fmt.Sprintf("%s/cookbooks", analyzeCacheDir))
```

Comment:
oops, this might have been a bug, we need to use `cookbooksDir` instead of this path, fixing.